test/unit/verifier/test_testinfra.py

```diff
@@ -312,7 +312,7 @@ def test_execute_bakes(patched_run_command, _patched_testinfra_get_tests,
 def test_executes_catches_and_exits_return_code(
         patched_run_command, _patched_testinfra_get_tests, _instance):
     patched_run_command.side_effect = sh.ErrorReturnCode_1(
-        sh.testinfra, b'', b'')
+        sh.Command('pytest'), b'', b'')
```

Comment:
why?

Reply:
`testinfra` is invoked through `pytest` AFAICT

Reply:
hm... so what's the reason behind this change anyway?

Reply:
The test was failing with "I can't find testinfra" from `sh`.

Reply:
The `testinfra` command line script was [dropped in 2.0.0](https://github.com/philpep/testinfra/commit/baab30ed1b39e92ce24ccad42a95e3be749c7245#diff-2eeaed663bd0d25b7e608891384b7298). The change here is correct. However, this could also be written as `sh.pytest, b'', b'')` which is more inline with the [sh examples](https://amoffat.github.io/sh/), the `Command()` call is superflous.

Reply:
OK, will patch to use same style invocation, thanks for the pointer!